src/library/basesqltablemodel.cpp

```diff
@@ -706,6 +706,9 @@ bool BaseSqlTableModel::setTrackValueForColumn(
         pTrack->setComment(value.toString());
     } else if (fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_BPM) == column) {
         pTrack->trySetBpm(static_cast<double>(value.toDouble()));
+    } else if (fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_REPLAYGAIN) == column) {
+        // pTrack->trySet(static_cast<double>(value.toDouble()));
```

Comment:
Please do not leave commented out code in when publishing a PR.

Reply:
The is the place where parsing of input values could be done. Changes are only accepted and forwarded to the `Track` object if successful.

Reply:
> Please do not leave commented out code in when publishing a PR.

oversight